src/s2wasm-main.cpp

```diff
@@ -46,6 +46,11 @@ int main(int argc, const char *argv[]) {
            [](Options *o, const std::string &argument) {
              o->extra["global-base"] = argument;
            })
+      .add("--stack-size", "-s", "Size of the user stack in linear memory",
```

Comment:
as mentioned earlier, the name seems wrong to me, if (1) we create the stack pointer anyhow, and (2) the stack might be created by a runtime later. Instead, how about `--create-stack` or `--create-stack-segment`? The last seems most precise, but maybe long.